src/api/v3/imprint.py

```diff
@@ -58,5 +60,10 @@ def imprint(request, region_slug, language_code):
         imprint_translation = region.imprint.get_public_translation(language_code)
         if imprint_translation:
             return JsonResponse(transform_imprint(imprint_translation))
+        if region.default_language:
+		    imprint_default_translation = region.imprint.get_public_translation(
+	            region.default_language.code
+        	)
+        	return JsonResponse(transform_imprint(imprint_default_translation))
```

Comment:
Even if the default language exists, we don't know if there's a public translation of the imprint in the default language.
Thus, I think we should also check this here (`transform_imprint` will throw an error when called with the argument `None`).

Reply:
Yes, we found this 2 seconds after the push :D